src/Symfony/Component/Process/Messenger/RunProcessFailedException.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Messenger;
```

Comment:
Should this be under the `Process\Exception` namespace?

Reply:
I think this is quite common in the codebase, so I think yes